src/core/sys/linux/sys/auxv.d

```diff
@@ -0,0 +1,165 @@
+/**
+ * D header file for GNU/Linux.
+ *
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Kai Nacke
+ */
+module core.sys.linux.sys.auxv;
+
+import core.sys.posix.sys.types;
+
+version (linux):
+extern (C):
+@system:
+nothrow:
+
+c_ulong getauxval(c_ulong type);
```

Comment:
Since this is the only function declared in the module, perhaps it's best to not use the : syntax for the attributes, but apply them directly to the function.